template/nim/flake.nix

```
@@ -0,0 +1,33 @@
+{
+  description = "Kickstart Nim package flake.";
+
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
```

```suggestion
  inputs = {
    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
    flake-parts.url = "github:hercules-ci/flake-parts";
  };
```

The CI

https://github.com/ALT-F4-LLC/kickstart.nix/actions/runs/13526859137/job/37799542801#step:6:128

is still not catching the divergence from

https://github.com/hercules-ci/flake-parts/pull/210 / #97 .

Since you aren't vendoring the template lockfiles, you could fix this adding a command like https://github.com/ALT-F4-LLC/kickstart.nix/pull/98/files#diff-f8cda1c916b88b1abf7b83d244e870087c5774a027b1938e91824da4c9db04e0R28-R29 from #98

```sh
nix flake update --no-use-registries path/to/temp/copy/of/flake/template
```

with the additional syntax to point to the correct directory containing the temporary copy of the template at

https://github.com/SecretVal/kickstart.nix/blob/6b348b22de70ee045cebc9ba5259dd68fa34bcc7/.github/workflows/flake-language.yml#L45

---

Alright. So if I explicitly add the flake-parts input there would be no problem?

---

Yes you could submit another PR with the suggestion above to make the new nim template like all of those in #97 .

---

Ok I will do that thanks